helm.tf

```diff
@@ -15,13 +15,15 @@ resource "helm_release" "ingress_nginx" {
   ]
 
   dynamic "set" {
+    for_each = var.helm_prometheus_enabled ? ["do it"] : []
```

Comment:
This isn't needed - if `var.helm_prometheus_enabled == false` then the helm value is overwritten with the default value (which is also false).

Behavior is the same, just one line less...